install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -52,6 +52,9 @@ properties:
             name:
               title: Name is the descriptive name of the port
               type: string
+            range:
```

Comment:
Should this default to some value? "default" maybe is the default port range? (I think that's what we had in the initial ticket?)

Reply:
I left it empty for the default case to make it backwards compatible for existing installations. Having an empty default also allows for it to be omitted if not being used. Happy to defer here.

Reply:
Unless we can find a way forward for detecting Autopilot with helm, `range` needs to be empty to know that someone is trying to use an unsupported feature. I don't love that.